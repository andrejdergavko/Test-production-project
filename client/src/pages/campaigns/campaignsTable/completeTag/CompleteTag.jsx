// @flow
import React from "react";
import { Tag } from "baseui/tag";

type CompleteTagT = {
  isActive: boolean,
};

function CompleteTag({ isActive }: CompleteTagT) {
  return (
    <Tag
      closeable={false}
      kind={isActive ? "positive" : "primary"}
      overrides={{
        Root: {
          style: () => ({
            marginLeft: "-7px",
          }),
        },
      }}
    >
      {isActive ? "Active" : "Inactive"}
    </Tag>
  );
}

export default CompleteTag;
