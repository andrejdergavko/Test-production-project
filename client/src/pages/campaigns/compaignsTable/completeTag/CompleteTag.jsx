import React from 'react';
import { Tag } from "baseui/tag";

function CompleteTag({ isActive }) {
	return (
		<Tag
			closeable={false}
			kind={isActive ? "positive" : "negative"}
			overrides={{
				Root: {
					style: ({ $theme }) => {
						return { marginLeft: "-7px" };
					},
				}
			}}
		>
			{isActive ? "Complete" : "Uncompleted"}
		</Tag>
	);
}

export default CompleteTag;