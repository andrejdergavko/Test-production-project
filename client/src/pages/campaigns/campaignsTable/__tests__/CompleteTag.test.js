import React from "react";
import renderer from "react-test-renderer";

import CompleteTag from "../completeTag";

describe("CompleteTag component:", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<CompleteTag />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
