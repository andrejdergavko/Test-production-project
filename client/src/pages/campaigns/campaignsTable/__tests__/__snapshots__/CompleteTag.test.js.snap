// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompleteTag component: renders correctly 1`] = `
<span
  aria-disabled={null}
  aria-label={null}
  className=""
  data-baseweb="tag"
  onBlur={[Function]}
  onClick={null}
  onFocus={[Function]}
  onKeyDown={[Function]}
  role={null}
  tabIndex={null}
>
  <span
    className=""
    title="Inactive"
  >
    Inactive
  </span>
</span>
`;
